monkey/infection_monkey/credential_store/credentials_store.py

```diff
@@ -0,0 +1,29 @@
+from typing import Mapping
+
+from .i_credentials_store import ICredentialsStore
+
+
+class CredentialsStore(ICredentialsStore):
+    def __init__(self, credentials: Mapping = None):
+        self.stored_credentials = credentials
```

Comment:
I don't see a reason for the constructor to accept credentials as an argument. It should setup whatever data structures are needed internally to store credentials and outside callers should use `add_credentials()` in order to add credentials.